.gitignore

```diff
@@ -7,3 +7,5 @@ npm-debug.log
 .samples.config.js*
 testenv
 okta-oidc-tck
+# matches pattern of other samples
+package-lock.json
```

Comment:
other sample repos do not track the `package-lock.json` so anyone that pulls the sample get the most recent versions of everything